e2e/kube/examples_test.go

```diff
@@ -66,6 +66,43 @@ var _ = Describe("Examples", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 
+			It("extended-job delete example must work", func() {
```

Comment:
@rohitsakala Saw e2e failed at this case

Reply:
Solved it.